umbraco/Umbraco-CMS
# Packages updated to show an angular view after install instead of ASCX (and more)

fixes #3996 

includes new 'package options' so you can go back to a packages options screen after it's installed.

**Make sure that #4010 is reviewed and merged first!!**

testing:

* Create a package in the back office that includes an options view (the UI lets you pick a file)
* Install the package - you will get redirected to the options view
* Since your package contains an options view you'll get a view 'package options' button when viewing the list of installed packages
* Create a package without a view and install it, you will just get redirected to the installed packages screen and you won't see a package options button
* also notice that the packaging route urls are now much much nicer
